scholarly/_navigator.py

```diff
@@ -156,6 +172,11 @@ def _use_proxy(self, http: str, https: str):
         if self._proxy_works:
             self.proxies = proxies
 
+        return {
+            "proxy_works": self._proxy_works,
+            "proxies": self.proxies,
```

Comment:
Why return self.proxies if we set it on line 173?
I would suggest that we return a bool if the setup was successful.

Also please add the remainder of the docstring as the method return has changed.